soc/nxp/mcx/mcxn23x/pinctrl_soc.h

```diff
@@ -16,23 +16,22 @@ extern "C" {
 
 typedef uint32_t pinctrl_soc_pin_t;
 
-#define Z_PINCTRL_MCX_PINCFG(node_id)					\
-	(PORT_PCR_DSE(DT_ENUM_IDX(node_id, drive_strength)) |			\
-	PORT_PCR_PS(DT_PROP(node_id, bias_pull_up)) |				\
-	PORT_PCR_PE(DT_PROP(node_id, bias_pull_up)) |				\
-	PORT_PCR_PE(DT_PROP(node_id, bias_pull_down)) |				\
-	PORT_PCR_ODE(DT_PROP(node_id, drive_open_drain)) |			\
-	PORT_PCR_SRE(DT_ENUM_IDX(node_id, slew_rate)) |				\
-	PORT_PCR_IBE(DT_PROP(node_id, input_enable)) |				\
+#define Z_PINCTRL_MCX_PINCFG(node_id)                     \
```

Comment:
Indentation still looks wrong here in github UI- maybe check your editor settings? Usually the easiest way to get these right is to indent the `\` characters with spaces instead of tabs (which CI won't complain about, if I remember right)